db/models/Snippet.js

```diff
@@ -7,7 +7,7 @@ const snippetSchema = new Schema(
     name: { type: String, required: true },
     code: { type: String, required: true },
     description: { type: String },
-    link: { type: String },
+    links: [Object],
```

Comment:
do the links appear properly in your database or do you get an error?
When I looked at your app locally, I received and error and adapted the Schema to this:

```suggestion
links: [
      {
        id: { type: String },
        value: { type: String },
      },
    ],
```

Reply:
In our database there seems to be no problem. Maybe it is because both our open branches use different schemas? Or can you tell us what exactly the error message was?

Reply:
I got a validation error, which indicated to me that something was amiss in the schema.

However, I just tried it out again and could not reproduce the error a second time.

It might be that your assumption is correct and it's due to the branches using different schemas.

In any case, since I cannot reproduce the issue anymore and there are no errors on your end, you can ignore my suggestion 😬